graphs/strongly_connected_components.py

```diff
@@ -0,0 +1,117 @@
+"""
+https://en.wikipedia.org/wiki/Strongly_connected_component
+
+Finding strongly connected components in directed graph
+
+"""
+
+test_graph_1 = {
```

Comment:
@cclauss Do you have any idea how we can avoid declaring these variables here but keep current doctests?

Reply:
I think we are OK here.  Naming helps a lot so when I see `test_graph_1`, I immediately know that this data is for testing purposes.  If I was going to move this algorithm to some larger program, I will need to do some level of refactoring.  I might move all the doctests and the test data to a separate test_xyz.py file or I might remove them altogether.  For our purposes in this repo, a single file that contains the algorithm, tests, and test data is good for learning purposes as long as we are diligent with our naming.